e2e-tests/ts-client/app/src/MockComponent.tsx

```diff
@@ -125,29 +125,29 @@ export const MockComponent = ({ webrtc }: Props) => {
         Update metadata on heart track
       </button>
       <div>
-        <VideoPlayer stream={heartMock.stream} />
+        <VideoPlayer stream={heartMock.stream}/>
```

Comment:
were there some formatter changes? or some other inconsistencies?

Reply:
Formatter does not work in this package. I reverted these changes